From daab5a37544e5c82926013fc69bfde6c4632456e Mon Sep 17 00:00:00 2001
From: Achill Gilgenast <achill@achill.org>
Date: Wed, 10 Dec 2025 20:39:22 +0100
Subject: [PATCH] Add systemd user service (#322)

* Add systemd user service

Adding a canonical systemd service to use on user sessions for starting
and supervising udiskie.

Also allows, for example, downstream distributions to package the systemd
service maintained upstream.
---
 example/udiskie.service | 9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 example/udiskie.service

diff --git a/example/udiskie.service b/example/udiskie.service
new file mode 100644
index 0000000..d4287b6
--- /dev/null
+++ b/example/udiskie.service
@@ -0,0 +1,9 @@
+[Unit]
+Description=Automounter for removable media
+
+[Service]
+ExecStart=/usr/bin/udiskie
+Type=simple
+
+[Install]
+WantedBy=graphical-session.target
